src/reporter/delete.py

```diff
@@ -34,11 +34,19 @@ def delete_entity(entity_id, type_=None, from_date=None, to_date=None):
 def delete_entities(entity_type, from_date=None, to_date=None,
                     drop_table=False):
     with translator_for(fiware_s()) as trans:
-        if drop_table:
-            trans.drop_table(etype=entity_type, fiware_service=fiware_s())
-            logging.getLogger(__name__).info(
-                "dropped entity_type {}".format(entity_type))
-            return 'entity table dropped', 204
+        if fiware_sp() is not None:
```

Comment:
@c0c0n3 rather than the list of service paths in the request, should we drop the table if and only if

request header include a single service path and that service path is the only service path available in the table?

the rational is that with the current code logic, it seems to me you can drop a whole table despite it includes more than one service path.